pkg/printers/humanreadable.go

```diff
@@ -442,11 +443,15 @@ func DecorateTable(table *metav1beta1.Table, options PrintOptions) error {
 			}
 
 			var m metav1.Object
-			if obj := row.Object.Object; obj != nil {
-				if acc, err := meta.Accessor(obj); err == nil {
-					m = acc
-				}
+			metaData := &metav1beta1.PartialObjectMetadata{}
```

Comment:
partialobject metadata is only included in certain table responses from the server. This needs to work with all cases.

A more consistent solution should be for the server to always include the "namespace" column in its columnDefinitions

Reply:
@juanvallejo always include the "namespace" for the server is  the solution I figure out in the first place.But I don't think it is best solution.Since currently, function below controls which fields 
the table should include. We can see that the param is fix. So if we modify this , is there any chance that we could influence other invoker? 
https://github.com/kubernetes/kubernetes/blob/a5c3c8d16c74d1fb710a88497f970ef6045f90d4/pkg/printers/storage/storage.go#L30

And for deal with all cases from the server,as far as I can see, code below contains all the possible type the table response may include. 
https://github.com/kubernetes/kubernetes/blob/a5c3c8d16c74d1fb710a88497f970ef6045f90d4/staging/src/k8s.io/apiserver/pkg/endpoints/handlers/response.go#L107